library/validation/LBF_Validation.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class LBF_Validation{
+
+    /*If another library is used the key names can be modified here*/
+    const VJS_KEY_REQUIRED = 'presence';
+    /*
+ * Function to generate the constraints used in validation.js library
+ * Using the data save in layout options validation
+ */
+    public static function generate_validate_constraints($form_id){
+
+
+        $fres = sqlStatement(
+             "SELECT layout_options.*,list_options.notes as validation_json 
+              FROM layout_options  
+              LEFT JOIN list_options ON layout_options.validation=list_options.option_id
```

Comment:
We need to  the layout_options to the list_options. in order to get the "rule" that is inside the notes of the list table.
list_options.list_id this is the foreign key we are inserting in the lbf layouts.

So i'm not sure what you meant in your question, can you explain it please?